tests/meshService.test.js

```diff
@@ -0,0 +1,128 @@
+import fs from "fs";
+import csvParser from "csv-parser";
+import dotenv from "dotenv";
+import payload from "../src/model/payload.js";
+import loader from "../src/service/loader.js";
+import senderService from "../src/service/senderService.js";
+import receiverService from "../src/service/receiverService.js"
+import meshService from "../src/service/MeshService.js";
+import { emptyDir } from "../src/service/Helper.js";
+
+// Empty input and output dirs
+async function emptyDirs() {
+  await emptyDir("./input")
+  await emptyDir("./output")
+}
+
+// Function to get current filenames
+function getFilenames(__dirname) {
+  let files = [];
+  fs.readdirSync(__dirname).forEach(file => {
+    files.push(file);
+  });
+  return files;
+}
+
+describe('mesh service', () => {
+
+  // Load variables
+  const loaderInstance = new loader(dotenv);
+
+  const messageFile =
+    process.env.MESH_DATA_FILE || "./tests/testdata-organizations-100000.csv";
+
+  // Create payload
+  const data = new payload(
```

Comment:
Class is now Payload not payload, please check other classes you've renamed as well and retest before confirming PR is ready for review again.